docs/3.process/7.connecting.md

```diff
@@ -0,0 +1,223 @@
+
+Once you have your service accounts and passwords you can proceed with
+the service configuration. What you have to do is to change the Keystone
+service to which they are validating the tokens. Currently your
+OpenStack services are connected to your local Keystone. You have to
+connect them to the FIWARE Lab Keystone.
+
+To do so, you have to modify the configuration files of each service
+changing the following parameters:
+
+-   `auth_uri`: You have to update it with the authorization endpoint
+    administrators have provided you when registering your account (in
+    the step [Creating your admin and services accounts](#accounts))
+
+-   `admin_tenant_name`: *service*
+
+-   `admin_user`: the username of this service account
+
+-   `admin_password`: the password of this service account
+
+Below is a list of configuration files that need to be changed:
+
+-   /etc/nova/nova.conf
+
+-   /etc/neutron/dhcp_agent.ini
+
+-   /etc/neutron/metadata_agent.ini
+
+-   /etc/neutron/neutron.conf
+
+-   /etc/neutron/l3_agent.ini
+
+-   /etc/neutron/api-paste.ini
+
+-   /etc/glance/glance-api.conf
+
+-   /etc/glance/glance-registry.conf
+
+-   /etc/glance/glance-cache.conf
+
+-   /etc/cinder/cinder.conf
+
+-   /etc/swift/proxy-server.conf
+
+Some of these files can be also hosted in the Controller and in the
+Compute nodes. It depends on the OpenStack configuration adopted by each
+Infrastructure Owner. For example, you can install OpenStack Networking
+in the Controller server or in a dedicated one. Usually in Compute nodes
+you have only to change the nova configuration file.
+
+Once these files are updated you should restart the corresponding
+services.
+
+#### Validating the registration
+
+Once all the steps are done, you have to check that the connection with
+the new Keystone is working properly. To do that you have to use the CLI
+tools to check:
+
+-   you can deploy a virtual machine
+
+-   you can deploy an image (deploy an image is equal to register as
+    image in glance)
+
+-   you can deploy a network
+
+-   you can attach a floating IP to a VM and it is reachable from
+    outside
+
+Keep in mind that if we want to use CLI tools, some environment
+variables must be first exported:
+
+```
+export OS_REGION_NAME="you_region_name"
+
+export OS_USERNAME=”your_user_name”
+
+export OS_PASSWORD=”your_password”
+
+export OS_AUTH_URL=http://cloud.lab.fiware.org:4730/v3
+
+export OS_PROJECT_NAME=admin
+
+export OS_PROJECT_DOMAIN_ID=default
+
+export OS_USER_DOMAIN_ID=default
+
+export OS_IDENTITY_API_VERSION=3
+```
+
+### Node publication in FIWARE Lab
+
+Once the registration process is finished and you have correctly
+validated it using the CLI tools, your node is ready for publication in
+FIWARE Lab.
+
+To publish it you have to contact the FIWARE Lab Keystone administrators
+telling them you have finished the process. They will check everything
+is ok and the will activate your node in the FIWARE Lab Cloud Portal and
```

Comment:
typo: they

Reply:
May be useful to have in this section the exact pieces of information the FIWARE Lab admins will need from the new Node, if known at this time of writing of course :)